addons/digest/views/digest_views.xml

```diff
@@ -56,8 +56,7 @@
                                 <group name="kpi_sales"/>
                                 <group name="custom" string="Custom" groups="base.group_system">
                                     <p>Want to add your own KPIs?<br />
-                                    <!-- TODO: change to actual documentation link!!! -->
-                                    <a href="https://www.odoo.com/documentation/" target="_blank"><i class="fa fa-arrow-right"></i> Check our Documentation</a></p>
+                                    <a href="https://www.odoo.com/documentation/saas-15.1/applications/general/digest_emails.html#custom-digest-emails" target="_blank"><i class="fa fa-arrow-right"></i> Check our Documentation</a></p>
```

Comment:
@fbarbe00 This should be 15.2. See https://github.com/odoo/odoo/pull/87178#discussion_r835231212